clients/src/main/java/org/apache/kafka/common/errors/InvalidMetadataException.java

```diff
@@ -19,7 +19,7 @@
 /**
  * An exception that may indicate the client's metadata is out of date
  */
-public abstract class InvalidMetadataException extends RetriableException {
+public class InvalidMetadataException extends RetriableException {
```

Comment:
Seems this was not really necessary since we just needed a sentinel value? Note that `common/errors` is a public package.

Reply:
I can use NetworkException instead (which is a subclass of InvalidMetadataException)